src/sccz80/define.h

```diff
@@ -130,6 +130,7 @@ struct type_s {
         int   params_offset;
         uint8_t  shortcall_rst;
         uint16_t shortcall_value;
+        char  shortcall_hl;
```

Comment:
I think adding an extra enum to symbol_flags is the best way to do this (and then obviously the change ripples through)